rules/splice.bzl

```diff
@@ -70,6 +70,7 @@ def _bitstream_splice_impl(ctx):
                 "--rcfile=",
                 "--logging",
                 "info",
```

Comment:
This is a nit, and not even your error, but would you mind changing this to `--logging=info`?